Xamarin.Forms.Platform.iOS/Renderers/ShellPageRendererTracker.cs

```diff
@@ -200,7 +200,7 @@ protected virtual async Task UpdateToolbarItems()
 			List<UIBarButtonItem> items = new List<UIBarButtonItem>();
 			if (Page != null)
 			{
-				foreach (var item in Page.ToolbarItems)
+				foreach (var item in System.Linq.Enumerable.OrderBy(Page.ToolbarItems, x=> x.Priority))
```

Comment:
Not important but we try not to add Linq to the mix :p

Reply:
Yeah, is there another way?

Reply:
Yea I stared at this for a little bit. I don't want to sort the list in place so I'd have to loop over it and allocate a new sorted list or do something like

Page.ToolBarItems.ToList().Sort(new IComparable())

Are we thinking that would be more efficient than Linq for a single digit list?